### ISSUE TITLE
Growing our own luffas 

### ISSUE THREAD

**Issue by iteles** [dwyl/home#116]
We don't have a huge amount of garden space but I would love to [take a page out of the National Trust's book](https://www.nationaltrust.org.uk/press-release/national-trust-staff-harvest-first-crop-of-grow-your-own-kitchen-sponges) and reserve a sunny section of the garden to experiment with growing our own luffa sponges for use in the kitchen and to offer to people for use in the shower.

When they're past their best, we can just compost them 😊 

This Good Housekeeping article is a start: https://www.goodhousekeeping.com/home/gardening/a20706975/how-to-grow-your-own-loofah-sponge/

**Comment by nelsonic** [comment]
Down for this idea. 💡 
Seeds ordered ✅ 
https://www.ebay.co.uk/itm/8-Seeds-Loofah-Sponge-Plant-Loofah-Cylindrica/183889052884
![image](https://user-images.githubusercontent.com/194400/73849695-40b65300-4822-11ea-8088-5678b381cd2d.png)
![image](https://user-images.githubusercontent.com/194400/73850139-0bf6cb80-4823-11ea-9279-8ea507e0e415.png)

As soon as they arrive we will figure out where to plant them. 👍

**Comment by iteles** [comment]
😄 I also had seeds in my cart and was just about to buy them when I got your comment notification @nelsonic 😆 

Also, they're _vines_ 🙌 🎉

**Comment by nelsonic** [comment]
@iteles yes, they are vines but they are _annual_ (_i.e. the vine **dies** at the end of the season_) 
unlike Kiwis or Passion Fruit which are _perennial_ (_will give fruits each year with minimal maintenance_).
I'm super up for growing the Luffas. (_but we will have a job to cut down the fine at the end of the year..._)

**Comment by iteles** [comment]
A very good point. Accessibility of the location will be key when we decide where they go.
